pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1356,5 +1356,49 @@
     <StablePackage Include="System.Runtime.Serialization.Xml">
       <Version>4.1.2</Version>
     </StablePackage>
+
+    <!-- Servicing 1.0.5 wave -->
+    <StablePackage Include="Microsoft.NETCore.Targets">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http">
+      <Version>4.1.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http.WebHttpHandler">
+      <Version>4.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Security">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.WebSockets.Client">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Text.Encodings.Web">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.debian.8-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.fedora.23-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.native.System.Security.Cryptography">
+      <Version>4.0.1</Version>
```

Comment:
On the one hand, I'm confused as to why the version for the wrapper package is .1 when the individual ones are .2.  But on the other hand, I'll believe it's correct.